README.md

```diff
@@ -45,6 +45,7 @@ The script can be configured via the following environment variables (called par
 | Parameter                           | Description | Default |
 |-------------------------------------|-------------|---------|
 | `TENOR_API_KEY`                     | Сlient key for privileged API access. This is the only **mandatory** parameter. | |
+| `TENOR_BLACKLIST` | A comma separated list of the GIFs ids which will be excluded when choosing one from the list returned by Tenor. If the script randomly chooses a GIF from the blacklist, the script sends one more request to choose a different GIF. | [641ee5344bdc3f9f4d3ef52344dfe6bd](https://media.tenor.com/images/641ee5344bdc3f9f4d3ef52344dfe6bd/tenor.gif) |
```

Comment:
If the script randomly chooses a GIF from the respond which belongs to the blacklist, the script sends one more request to Tenor.